README.md

```diff
@@ -101,7 +101,7 @@ Ethereum common tests are created for all clients to test against. We plan to pr
    - [GeneralStateTests](https://github.com/ethereum/tests/tree/develop/GeneralStateTests) 1174/1180 = 99.5% passing
 - [x] Byzantium
    - [BlockchainTests](https://github.com/ethereum/tests/tree/develop/BlockchainTests) (Includes GeneralStateTests) 4778/4959 = 96.4% passing
-   - [GeneralStateTests](https://github.com/ethereum/tests/tree/develop/GeneralStateTests) 4564/4632= 98.5% passing
+   - [GeneralStateTests](https://github.com/ethereum/tests/tree/develop/GeneralStateTests) 4574/4754= 96.2% passing
```

Comment:
Interesting, the percentage went down but the number of tests increased? Did we resync recently? Was it part of this work?

Reply:
We must've resynced recently, and the README wasn't updated. The resync wasn't part of this PR. 

But I ran the `generate_state_tests` task and it showed different numbers. It's hard to tell from just looking at this if more or less tests passed because of the changes in this PR. I can split this into a separate PR if it makes more sense.

Reply:
Decided against updating this since no tests passed. 082d0ecc1323c431751c4768d4d4a5e5499caf0f